### PR TITLE
Travis changes

### PR DESCRIPTION
**Pull request by Naxodile**
Makes travis check other maps again.

### CODE REVIEW

File: maps/glloydstation/glloydstation.dm [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #if !defined(USING_MAP_DATUM)
 
+	#include "../../code/modules/urist/area&turf/areas.dm"
+	#include "../../code/modules/urist/machinery/misc.dm"
```
Comment (Glloyd) [inline maps/glloydstation/glloydstation.dm]: put this file back where it was. Move the camera definition in that file to glloydstation_presets .dm

Comment (scrdest) [inline maps/glloydstation/glloydstation.dm]: Relative pathing is bad practice, better to just move these files wholesale to shared/urist or something along these lines

**Comment by Glloyd** [comment]
yeah, having those relative paths there are iffy. You could just move the areas from that file to glloydstation_areas.dm and delete the old area file. That's what we should've just done anyways

**Comment by Glloyd** [comment]
and now its runtiming from the awaymaps on the baymaps. Is it even worth it having those run through Travis?

**Pull request closed without merging**